Makefile

```diff
@@ -99,7 +102,7 @@ helm-keylime-undeploy: ## Undeploy the keylime helm chart
 helm-keylime-deploy: ## Deploy the keylime helm chart
 	{ \
 	touch $(HELM_CHART_CUSTOM_VALUES);\
-	cat $(HACK_DIR)/k8s-poc/admin/keylime_tenant | sed -e "s/source/#source/g" -e "s/#export/export/g" -e "s/announce/echo/g" -e "s/REPLACE_KEYLIME_NAMESPACE/$(HELM_CHART_NAMESPACE)/g" -e "s^bin/kt^bin/keylime_tenant^g" > $(MKFILE_DIR)/kt;\
+	cat $(HACK_DIR)/k8s-poc/admin/kt | sed -e "s/#export/export/g" -e "s/REPLACE_KEYLIME_NAMESPACE/$(HELM_CHART_NAMESPACE)/g" > $(MKFILE_DIR)/kt;\
```

Comment:
Sorry, but I don't understand this ... where is this `$(HACK_DIR)/k8s-poc/admin/kt`? It does not seem to be in the repository ... might you have forgotten a `git mv  $(HACK_DIR)/k8s-poc/admin/keylime_tenant $(HACK_DIR)/k8s-poc/admin/kt`?

Reply:
Ah, a silly mistake. I have left `kt` in `. gitnore`. Fixed now, thanks